ref/Lists_of_Geometric_Elements.md

```diff
@@ -6,20 +6,35 @@ This is done with operators that return lists of elements of a particular kind.
 
 #### All points of the construction: `allpoints()`
 
+#### All points incident to an geometric object: `allpoints(‹geo›)`
```

Comment:
@strobelm Sorry to bother you again. It should be 'a geometric object', not 'an', in all instances.